index.html

```diff
@@ -0,0 +1,49 @@
+---
+layout: default
+
+link: Tell them to support 2FA
```

Comment:
Haha might want to change this. Thankfully they use MIT license so we can just link to them in a license file as a way to provide a link back

Reply:
Removed